ulib/FStar.Pervasives.fsti

```diff
@@ -868,3 +868,6 @@ val with_type (#t: Type) (e: t) : Tot t
     One of its uses is in types of layered effect combinators that
     are subjected to stricter typing discipline (no subtyping) *)
 unfold let eqtype_as_type (a:eqtype) : Type = a
+
+irreducible
```

Comment:
Can you please add a comment here on the semantics of the attribute (in the style of the other attributes in the file)?